### PR TITLE
Update OVMS to use 2022.3 tag

### PR DESCRIPTION
**Pull request by Xaenalt**
## Merge criteria:

- [X] The commits are squashed in a cohesive manner and have meaningful messages.
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by Xaenalt** [comment]
Should we use openvino/model_server or the access.redhat.com one?

**Comment by LaVLaS** [comment]
> Should we use openvino/model_server or the access.redhat.com one?

We should be using any image repo that does not require authenticated credentials

**Comment by LaVLaS** [comment]
@Xaenalt Looks like there is a test failure related to the dashboard. It should be resolved in #714

**Comment by LaVLaS** [comment]
@Xaenalt Can you rebase your PR to pull in the CI test fix?

**Comment by anishasthana** [comment]
@Xaenalt Can you raise a PR to the odh modelmesh serving repo as well to update the manifests in there?

File: model-mesh/runtimes/kustomization.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ images:
 
   - name: ovms-1
     newName: openvino/model_server
-    newTag: "2022.1"
+    newTag: "2022.3"
```
Comment (anishasthana) [inline model-mesh/runtimes/kustomization.yaml:30]: Does this change actually change what is created by odh-dashboard? @lucferbux fyi^^

**Review by anishasthana — APPROVED** [review]

**Pull request merged**